.prettierrc

```diff
@@ -1,5 +1,8 @@
 {
   "semi": true,
   "singleQuote": true,
-  "trailingComma": "es5"
-}
\ No newline at end of file
+  "trailingComma": "es5",
+  "printWidth": 120,
+  "bracketSpacing": true,
+  "tabWidth": 4
```

Comment:
This can be changed back to `2` if desired, just personal preference while working on it.

Reply:
I am in favour of 4 tabs. :slightly_smiling_face: